conans/tools.py

```diff
@@ -22,7 +22,8 @@
 from conans.errors import ConanException
 from conans.model.version import Version
 # noinspection PyUnresolvedReferences
-from conans.util.files import _generic_algorithm_sum, load, save, sha256sum, sha1sum, md5sum, md5
+from conans.util.files import _generic_algorithm_sum, load, save, sha256sum, \
+    sha1sum, md5sum, md5, touch, relative_dirs, rmdir, mkdir, path_exists
```

Comment:
I am not sure, ``path_exists`` and ``relative_dirs`` might be too much for conan internals, do you think they provide value for package creators?